src/garage/torch/algos/sac.py

```diff
@@ -94,7 +94,7 @@ def __init__(
         replay_buffer,
         *,  # Everything after this is numbers.
         max_episode_length,
```

Comment:
```suggestion
        max_episode_length_explore,
```

Reply:
if i don't set it, does it use the `max_episode_length` from the environment?

Reply:
no, it is default 1000. I had done this when we were only doing apples to apples comparisons with the paper. A less confusing behavior would be do use `max_episode_length_explore`

Reply:
i agree with that as long as `max_episode_length_explore` defaults to `env.spec.max_episode_length`

In general I think people shouldn't interact with these unless they have very specific needs (e.g. your baseline comparisons), e.g. this wouldn't appear in an example.